alipay/container-auto-tune
# Feature add web structure

# Which issue does this PR close?

Closes https://github.com/alipay/container-auto-tune/issues/22

# Rationale for this change
 - embed the tmaestro front-end into tmaestro-server, which can start front-end when tmaestro-server starts up.
 - remove tmaestro-front container from tmaestro-lite.yaml

it has two advantages:
- it can save the host resource  by avoiding to use a container to start front-end.
- it is more lightweight by embed into tmaestro-server, it can start up with java starts up.



# What changes are included in this PR?
![image](https://user-images.githubusercontent.com/39024249/201918657-9e7d60b3-d7c2-404b-8910-d02491370f0f.png)


# Are there any user-facing changes?
NO



# How does this change test
1. start the tmaestro-server following by README.
2. open url  [tmaestro-console](http://localhost:30081), you will see the tmaestro-front
![image](https://user-images.githubusercontent.com/39024249/201920406-b0b48bce-fe1a-4c12-9040-c912e57d6124.png)

![image](https://user-images.githubusercontent.com/39024249/201920810-0b0c3224-ac2a-411b-b01d-f0d15769672f.png)
